src/main/java/ch/njol/skript/expressions/ExprAI.java

```diff
@@ -50,12 +50,21 @@ public Boolean convert(LivingEntity entity) {
 	@Nullable
 	@Override
 	public Class<?>[] acceptChange(Changer.ChangeMode mode) {
-		return mode == Changer.ChangeMode.SET ? CollectionUtils.array(Boolean.class) : null;
+		switch (mode) {
+			case TOGGLE:
+			case SET:
+				return CollectionUtils.array(Boolean.class);
+			default:
+				return null;
+		}
 	}
 	
 	@Override
 	public void change(Event event, @Nullable Object[] delta, Changer.ChangeMode mode) {
 		if (delta == null || delta[0] == null) {
+			if(mode == Changer.ChangeMode.TOGGLE)
```

Comment:
There should be a space after the `if` statement. There should also be curly brackets around the `if` statement IMO